image/templates/helm/stackrox-central/README.md.htpl

```diff
@@ -35,30 +37,6 @@ helm install -n stackrox --create-namespace \
 If you have a copy of this chart on your machine, you can also reference the
```

Comment:
[Comment about the line above] I think it should be
```
helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services
```
Please check commands in README that we recently tried and tested with Jan. https://github.com/stackrox/stackrox/#quick-installation-via-helm

Similarly, please double-check `helm upgrade ...` command towards the end of this file.

Reply:
Checked and updated all Helm chart invocations